tests/.infra/centos-ci/rc/cico-rc-multiuser-happy-path-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+set -e
+set +x
+
+source tests/.infra/centos-ci/functional_tests_utils.sh
+
+function prepareCustomResourceFile() {
+  echo "======== Patch custom-resource.yaml ========"
+  cd /tmp
+  wget https://raw.githubusercontent.com/eclipse/che-operator/master/deploy/crds/org_v1_che_cr.yaml -O custom-resource.yaml
+  sed -i "s@server:@server:\n    customCheProperties:\n      CHE_LIMITS_USER_WORKSPACES_RUN_COUNT: '-1'@g" /tmp/custom-resource.yaml
+  sed -i "s/customCheProperties:/customCheProperties:\n      CHE_WORKSPACE_AGENT_DEV_INACTIVE__STOP__TIMEOUT__MS: '300000'/" /tmp/custom-resource.yaml
+  sed -i "s@identityProviderImage: 'quay.io/eclipse/che-keycloak:nightly'@identityProviderImage: 'quay.io/eclipse/che-keycloak:$RELEASE_TAG'@g" /tmp/custom-resource.yaml
+  sed -i "s@cheImage: ''@cheImage: 'quay.io/eclipse/che-server'@g" /tmp/custom-resource.yaml
+  sed -i "s@cheImageTag: 'nightly'@cheImageTag: '$RELEASE_TAG'@g" /tmp/custom-resource.yaml
+  sed -i "s@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:nightly'@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:$RELEASE_VERSION'@g" /tmp/custom-resource.yaml
+  sed -i "s@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:nightly'@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:$RELEASE_VERSION'@g " /tmp/custom-resource.yaml
+  cat /tmp/custom-resource.yaml
+}
+
+ # It needs:
+ # 1) to implement the patch of 'devfile' to appropriate release tags images
+ # 2) to save screenshots if the 'happy-path' is fail
+
+setupEnvs
+installKVM
+installDependencies
+setupReleaseVersionAndTag
+installAndStartMinishift
+loginToOpenshiftAndSetDevRole
+prepareCustomResourceFile
+installCheCtl
+deployCheIntoCluster  --che-operator-cr-yaml=/tmp/custom-resource.yaml
+createTestUserAndObtainUserToken
+createTestWorkspaceAndRunTest  --devfile=https://raw.githubusercontent.com/eclipse/che/cico-rc-test/tests/e2e/files/happy-path/happy-path-workspace.yaml
+getOpenshiftLogs
+archiveArtifacts "rc-multiuser-happy-path-test"
```

Comment:
If test fails the artifacts have not been stored. I fixed it here: https://github.com/eclipse/che/pull/16035/

Reply:
Actually the _rc-happy-path_ script requires additional update related to prepare _devfile_ to appropriate release version. I had conversation with @dmytro-ndp  about that, he already did it in _crw-ci-pre-release-testing_ job. But now this task has a low priority and it will be done later.

Reply:
And what? How it linked with my comment?

Reply:
I meant that really the script is not fully ready, the job will be failure, until. And I wasn't worried about save artefacts . Sure I'll fix  that.

Reply:
Done.